Area/src/Triangle.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by webonise on 14/9/16.
+ */
+public class Triangle extends Rectangle {
+
+    public Triangle(int wid,int high){
+        super(wid,high);
```

Comment:
wid is not proper variable name. We have followed different different naming like w,wid,width. Give appropriate name to all places.